src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_consts.py

```diff
@@ -7,11 +7,13 @@
 CONST_ACSTOR_IO_ENGINE_LABEL_KEY = "acstor.azure.com/io-engine"
 CONST_ACSTOR_IO_ENGINE_LABEL_VAL = "acstor"
 CONST_ACSTOR_K8S_EXTENSION_NAME = "microsoft.azurecontainerstorage"
+CONST_ACSTOR_V2_K8S_EXTENSION_NAME = "microsoft.acstortest"
 CONST_DISK_TYPE_EPHEMERAL_VOLUME_ONLY = "EphemeralVolumeOnly"
 CONST_DISK_TYPE_PV_WITH_ANNOTATION = "PersistentVolumeWithAnnotation"
 CONST_EPHEMERAL_NVME_PERF_TIER_BASIC = "Basic"
 CONST_EPHEMERAL_NVME_PERF_TIER_PREMIUM = "Premium"
 CONST_EPHEMERAL_NVME_PERF_TIER_STANDARD = "Standard"
+CONST_ACSTOR_V2_EXT_INSTALLATION_NAME = "storage"
```

Comment:
Note: in v1 this param determined the pod prefix. In v2, the prefix will always be "acstor" as it doesn't use the helm release name. You can change this to whatever is most suitable.

Reply:
Actually, it will be used in the `app.kubernetes.io/instance` label on resources. Can you set it to `acstor`?